packages/libs/react-components/src/components/Typography/styles.ts

```diff
@@ -131,7 +131,6 @@ export const transformVariant = {
 
 export const BaseText = styled('span', {
   fontWeight: '$regular',
-  color: '$foreground',
```

Comment:
What is the reason for removing this? I'm not sure if the theme switching will work in storybook without this

Reply:
doing it like this will have the color always the same color.
removing it, it will inherit the font color of the parent element.
This will still work fine with theme switching.

We can do `color: "inherit"`
maybe that is more clear?

Reply:
Ahhh maybe if you apply the color in global styles it would work. I'm fine with it being removed as well, I would just suggest to check that this still works in storybook in dark mode 👍

Reply:
I did.... and it does 👍🏻